src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -231,11 +228,13 @@ def _create_from_flow(
         node_override: Optional[Dict[str, Dict[str, Any]]] = None,
         line_timeout_sec: Optional[int] = None,
     ):
+        MultimediaProcessor.create(flow.message_format)
```

Comment:
When one flow calls another, it seems there would be a bug if two flows have different contracts.
We may add a todo to handle such scenarios.